web/src/components/PluginFlow/locales/en-US.ts

```diff
@@ -20,7 +20,7 @@ export default {
   'component.plugin-flow.text.condition2': 'Condition',
   'component.plugin-flow.text.condition.placeholder': 'Please enter the rule',
   'component.plugin-flow.text.without-data': 'Found node without configuration',
-  'component.plugin-flow.text.plugin-without-data.description': 'Please condigure plugin: ',
+  'component.plugin-flow.text.plugin-without-data.description': 'Please configure plugin: ',
```

Comment:
Weird, seems we had it fixed in PR [#2417](https://github.com/apache/apisix-dashboard/pull/2417/files#diff-13129ce937cf7473d201943d3e07e80c0cfdd9349cd235ee869f652b6d34054bR23), Any idea of What Happened ?